guardian/typerighter
# Add UI for Showing & Refreshing Rules

## What does this change?

Adds new UI in the rule-manager to Show and Refresh rules.

This will replace the existing UI in the Checker service. 

## How to test

Deploy, verify that all rules are available. The new UI uses an in-memory paginated view of the rules but also allows search. Search for text that appears in the Rules that you care about (we are searching across the pattern, id, description and type)

Try removing / adding a rule to the sheet and refreshing. Verify that the rule is added / removed as expected


## How can we measure success?

It works in the same way as the existing code in the checker UI

## Have we considered potential risks?

Think so?

## Images

![Screenshot 2023-03-28 at 17 09 27](https://user-images.githubusercontent.com/3277259/228302249-2f00d289-4741-4d36-82f7-31ce030a9ed8.png)
